code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -355,6 +355,16 @@
 	)
 	in_list_or_max = 1
 
+	attackby(obj/item/W as obj, mob/user as mob, obj/item/storage/T)
+		if (istype(W, /obj/item/storage/firstaid/regular))
+			var/obj/item/storage/S = W
+			for (var/obj/item/I in S.get_contents())
+				if (..(I, user, S) == 0)
```

Comment:
Just a bit cleanliness (checking for all false values instead of just `0` by negating the result of the `..()` call).
```suggestion
				if (!..(I, user, S))
```